build.gradle

```diff
@@ -67,12 +67,13 @@ ext {
     moduleName = "io.aiven.kafka.connect.http"
 
     avroVersion = "1.8.1" // Version 1.8.1 brings Jackson 1.9.x/org.codehaus.jackson package for Avro and Confluent Platform 4.1.4.
-    confluentPlatformVersion = "4.1.4" // For compatibility tests use version 4.1.4.
+    confluentPlatformVersion = "6.0.2" // For compatibility tests use version 6.0.2 to match Kafka 2.6.
+                                       // NOTE: Confluent Platform v6.0.3 has a dependency mismatch issue.
     hamcrestVersion = "2.2"
     jacksonVersion = "2.13.2" // This Jackson is used in the tests.
     jupiterVersion = "5.8.2"
-    kafkaVersion = "2.0.1" // Oldest version supported, new versions are backwards compatible.
-    jettyVersion = "9.4.11.v20180605"
+    kafkaVersion = '2.6.3'
+    jettyVersion = "9.4.45.v20220203"
```

Comment:
To match the rest of the transitive dependencies perhaps we should use `9.4.39.v20210325` here instead.